dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -194,7 +195,13 @@ def destroy
     def search
       email = params[:email]
       user = User.find_by_email email
-      filtered_applications = @applications.where(
+
+      # only workshop admins can see incomplete applications
+      filtered_applications = current_user.workshop_admin? ? @applications.where(
+        application_year: APPLICATION_CURRENT_YEAR,
+        application_type: [TEACHER_APPLICATION, FACILITATOR_APPLICATION],
+        user: user
+      ) : @applications.where.not(status: 'incomplete').where(
```

Comment:
optional: one idea to deduplicate here is to take advantage of the fact that active record queries do not get executed right away. so you can write this as:
```
      filtered_applications = current_user.workshop_admin? ? @applications.where(
        application_year: APPLICATION_CURRENT_YEAR,
        application_type: [TEACHER_APPLICATION, FACILITATOR_APPLICATION],
        user: user
      )
      filtered_applications = current_user.workshop_admin? ? filtered_applications : filtered_applications.where.not(status: 'incomplete')
```
to me this is a little clearer what the difference is depending on whether the user is a workshop admin, but I would only suggest updating if this way seems clearer to you too.

Reply:
Yes, this is great –– thanks!